examples/langchain/retrieval_qa_chain.py

```diff
@@ -9,6 +8,8 @@
 from langchain.text_splitter import CharacterTextSplitter
 from langchain.vectorstores import FAISS
 
+import mlflow
```

Comment:
Can we also add an example demonstrating how to user the MLflow retriever support?

Reply:
added